### PR TITLE
Ta i bruk nye endepunkter uten fnr som queryparam

### PR DESCRIPTION
**Pull request by JulieHillRoa**

### CODE REVIEW

File: src/api/veilarbvedtaksstotte/utkast.ts [hunk 1]
```diff
@@ -31,11 +31,11 @@ export function fetchTaOverUtkast(vedtakId: number): AxiosPromise {
 }
 
 export function lagNyttUtkast(fnr: string): AxiosPromise {
-	return axiosInstance.post(`${VEILARBVEDTAKSSTOTTE_API}/utkast`, { fnr });
+	return axiosInstance.post(`${VEILARBVEDTAKSSTOTTE_API}/v2/utkast`, { fnr });
```
Comment (slovrid) [inline src/api/veilarbvedtaksstotte/utkast.ts:34]: Endepunktet for henting og lagring er nå likt i og med at begge benytter `POST`. Det må vel fikses i den gitte API-appen.

Comment (JulieHillRoa) [reply src/api/veilarbvedtaksstotte/utkast.ts]: Skal nå være fikset :P

File: src/api/veilarbperson.ts [hunk 2]
```diff
@@ -11,5 +11,5 @@ export interface MalformData {
 }
 
 export function fetchMalform(fnr: string): AxiosPromise<MalformData> {
-	return axiosInstance.get(`/veilarbperson/api/v2/person/malform?fnr=${fnr}`);
+	return axiosInstance.post(`/veilarbperson/api/v3/person/hent-malform`, { fnr });
```
Comment (dragutin-nav) [inline src/api/veilarbperson.ts:14]: is it necessary to send body content as: `{fnr: fnr}`, or it works like this as well?

**Review by dragutin-nav — APPROVED** [review]

File: src/api/veilarbperson.ts (hunk 2 above)
Comment (JulieHillRoa) [reply src/api/veilarbperson.ts:14]: json is "a little to smart"
som when the key and the valuevariabel is the same: `{ fnr: fnr }` equals `{ fnr }` , but if one of them was different you have to specify: `{ fnr: fodselsnummer }`... if that makes sence

Comment (slovrid) [reply src/api/veilarbperson.ts:14]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions

**Review by slovrid — APPROVED** [review]

**Pull request merged**